client/lib/products-values/index.js

```diff
@@ -202,12 +203,7 @@ export function isJetpackMonthlyPlan( product ) {
 }
 
 export function isJetpackProduct( product ) {
-	const jetpackProducts = [
-		'jetpack_backup_daily',
-		'jetpack_backup_realtime',
-		'jetpack_backup_daily_monthly',
-		'jetpack_backup_realtime_monthly',
-	];
+	const jetpackProducts = [ ...JETPACK_BACKUP_PRODUCTS ];
```

Comment:
Note that we create a new array here to prepare for future additional products like Scan.